scripts/generate-config.sh

```diff
@@ -39,12 +39,21 @@ compare_extensions=(
   yaml
   go
   proto_descriptor
+  html
 )
 
+exit_with=0
+echo "Generating config against istio $(cd ${istio_src} && git describe --dirty --tags --abbrev=12)"
 for i in "${compare_extensions[@]}" ; do
   for file in "${target_dir}/config/"*.${i} ; do
-    diff -b "$file" "config/${file##*/}"
+    if [[ $(diff --brief "${file}" "${root_dir}/config/${file##*/}") ]];
+    then
+        exit_with=1
+        echo "Diff detected in file ${file}"
```

Comment:
Have you tried just letting the output of the diff command through (`--brief` does the same as your `echo` line)? It would allow you to simplify this like:
```shell
diff --brief file1 file2 || exit_with=1
```

Reply:
Well I have since you suggested it. Works nicely 👍 . Pushed